src/config/index.ts

```diff
@@ -53,10 +53,10 @@ export const FEED_GOCD_JOB_RUNNER_CHANNEL_ID = // #feed-gocd-job-runner
   process.env.FEED_GOCD_JOB_RUNNER_CHANNEL_ID || 'C07E8TG7VJP';
 export const KAFKA_CONTROL_PLANE_CHANNEL_ID = // #feed-topicctl
   process.env.KAFKA_CONTROL_PLANE_CHANNEL_ID || 'C07E9S96YPM';
-export const SUPPORT_CHANNEL_ID = // #discuss-support-open-source
+export const SUPPORT_CHANNEL_ID = // #discuss-support-github
   process.env.SUPPORT_CHANNEL_ID || 'C02KHRNRZ1B';
-export const TEAM_OSPO_CHANNEL_ID = // #team-ospo
-  process.env.TEAM_OSPO_CHANNEL_ID || 'G01F3FQ0T41';
+export const TEAM_DEV_INFRA_CHANNEL_ID = // #feed-dev-infra
+  process.env.TEAM_DEV_INFRA_CHANNEL_ID || 'C05816N2A2K';
```

Comment:
Can you confirm this @hubertdeng123? I think maybe this should be a private channel not the public feed one.

Reply:
```suggestion
export const TEAM_DEV_INFRA_CHANNEL_ID = // #team-dev-infra
  process.env.TEAM_DEV_INFRA_CHANNEL_ID || 'G01F13Y6AQ0';
```

Reply:
Will do, thanks!